src/coreclr/jit/copyprop.cpp

```diff
@@ -33,7 +33,7 @@ void Compiler::optBlockCopyPropPopStacks(BasicBlock* block, LclNumToLiveDefsMap*
         for (GenTree* const tree : stmt->TreeList())
         {
             GenTreeLclVarCommon* lclDefNode = nullptr;
-            if (tree->OperIs(GT_ASG) && tree->DefinesLocal(this, &lclDefNode))
+            if (tree->OperIsSsaDef() && tree->DefinesLocal(this, &lclDefNode))
```

Comment:
Did you check around to see if there are other instances of the "old pattern" of checking for SSA defs?

Reply:
Yes. I verified the places where we were checking for `DefinesLocal()`.